src/app/public/modules/radio/radio-group.component.spec.ts

```diff
@@ -303,6 +303,40 @@ describe('Radio group component', function () {
     expect(radios.item(1).checked).toEqual(false);
   }));
 
+  it('should support resetting form control when fields are added dynamically', fakeAsync(function () {
+    componentInstance.radioForm.patchValue({
+      option: componentInstance.options[0]
+    });
+
+    fixture.detectChanges();
+
+    expect(componentInstance.radioForm.value).toEqual({ option: { name: 'Lillith Corharvest', disabled: false } });
+
+    // Toggle the field's generation on and off to make sure the form control's state
+    // isn't directly tied to the template's change detection.
+    componentInstance.radioGroupEnabled = false;
+    fixture.detectChanges();
+    tick();
+
+    componentInstance.radioGroupEnabled = true;
+    fixture.detectChanges();
+    tick();
```

Comment:
should we be expecting anything after toggling enabled to true? Or maybe selecting one of the options? Regardless of resetting I wouldn't think this would ever have a value for `radioForm.value.option`

Reply:
Fair observation. I added another expectation against the value, to make sure nothing changes there.